skypeweb/skypeweb_login.c

```diff
@@ -369,7 +369,7 @@ skypeweb_login_did_got_api_skypetoken(PurpleHttpConnection *http_conn, PurpleHtt
 	}
 
 	purple_connection_error(sa->pc, error_type,
-		error ? error : _("Failed getting Skype Token (alt)"));
+		error ? error : _("Failed getting Skype Token (alt), please try logging in via browser first"));

```

Comment:
I don't believe the alternate login is resolved by logging in with the browser?

Reply:
Sorry. 
I believe you know better than me :)
Changing my comit